pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/InTransformFunction.java

```diff
@@ -199,6 +210,8 @@ public int[] transformToIntValuesSV(ValueBlock valueBlock) {
               }
             }
             break;
+          case UNKNOWN:
+            break;
```

Comment:
Maybe put default null value in _intValuesSV to be consistent across calls

Reply:
If the value is null, we don't care about the values in _intValuesSV so we don't need to set the values.

So we want to keep here as is and remove the code elsewhere that sets the default null value in _intValuesSV ([e.g.](https://github.com/apache/pinot/blob/60ee5eb0e8a2cf8276565ae8b20fb81292315843/pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunction.java#L173)) and the related unit tests ([e.g.](https://github.com/apache/pinot/blob/d50f9ee8b04f615abc57eab4c08a53ad4902f72a/pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/ScalarTransformFunctionWrapperTest.java#L883))? I'd be happy to do the cleanup in a different PR.

Reply:
My point is to make the result consistent when enableNullHandling is set to false. In that call, we don't get the nullbitmap.

Reply:
We don't have the problem of inconsistent results.

We can enable/disable null in two places:
- Data ingestion/storage 
- Query option 

If we disable null in data ingestion/storage, there is no inconsistency problem in the results between null enabled and disabled in the query option. If we enable null in data ingestion/storage, we should only allow the null enabled query option.

Reply:
If we enable null in data ingestion/storage, it is possible client will also call the query without  null enabled query option. and the query should give consistent results. We decided not to crush the query per discussion with @Jackie-Jiang

Reply:
Modified the code to put default null value in `_intValuesSV`.

Had an offline discussion with @61yao:
In theory if we enable null in data ingestion/storage and the client makes a query without the null enabled query option, we should fail the query. In practice we don't fail the query.